plutus-use-cases/src/Language/PlutusTx/Coordination/Contracts/PubKey.hs

```diff
@@ -0,0 +1,79 @@
+{-# LANGUAGE FlexibleContexts    #-}
+{-# LANGUAGE DataKinds           #-}
+{-# LANGUAGE DeriveGeneric       #-}
+{-# LANGUAGE OverloadedStrings   #-}
+{-# LANGUAGE RecordWildCards     #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TemplateHaskell     #-}
+-- | A "pay-to-pubkey" transaction output implemented as a Plutus
+--   contract. This is useful if you need something that behaves like
+--   a pay-to-pubkey output, but is not (easily) identified by wallets
+--   as one.
+module Language.PlutusTx.Coordination.Contracts.PubKey(
+      pkValidator
+    , pkRedeemer
+    , pkDataScript
+    , lock
+    , spend
+    ) where
+
+import           Data.Maybe (listToMaybe)
+import qualified Data.Map   as Map
+import qualified Data.Text  as Text
+
+import qualified Language.PlutusTx            as P
+import           Ledger                       as Ledger hiding (initialise, to)
+import           Ledger.Validation            as V
+import           Wallet.API                   as WAPI
+
+pkValidator :: PubKey -> ValidatorScript

```

Comment:
This seems like it should be able to share some code with the multisig contract, perhaps?

Reply:
Are you referring to the validator or to the off-chain code?

Reply:
The validator - it's just remembering some signatures and checking there is one. In general this is multi-sig where n and m = 1. Not sure if it helps to write it that way, but less code is better...